README.md

```diff
@@ -28,6 +28,7 @@ Templates for [Appwrite](https://appwrite.io/) Functions. These templates can be
 | Slack Command Bot           | [✅](node/slack-command-bot)           | 🏗️                               | 🏗️                            | 🏗️                            | 🏗️                                        | 🏗️                            | 🏗️                             | 🏗️                             | 🏗️                               | 🏗️              | 🏗️                 | 🏗️               | 🏗️                |
 | Storage Cleaner             | [✅](node/storage-cleaner)             | 🏗️                               | 🏗️                            | 🏗️                            | 🏗️                                        | 🏗️                            | 🏗️                             | 🏗️                             | 🏗️                               | 🏗️              | 🏗️                 | 🏗️               | 🏗️                |
 | Subscriptions with Stripe   | [✅](node/subscriptions-with-stripe)   | 🏗️                               | 🏗️                            | 🏗️                            | 🏗️                                        | 🏗️                            | 🏗️                             | 🏗️                             | 🏗️                               | 🏗️              | 🏗️                 | 🏗️               | 🏗️                |
+| Translate with Huggingface  | [✅](node/translate-with-huggingface)  | 🏗️                               | 🏗️                            | 🏗️                            | 🏗️                                        | 🏗️                            | 🏗️                             | 🏗️                             | 🏗️                               | 🏗️              | 🏗️                 | 🏗️               | 🏗️                |
```

Comment:
this path /translation-with-huggingface does not match the folder name used below.
Also I think we should rename the folder and the package name to `huggingface-language-translation`